app/services/api/v1x0/catalog/copy_portfolio_item.rb

```diff
@@ -42,6 +42,8 @@ def make_copy
               new_portfolio_item.service_plans << new_plan
             end
 
+            new_portfolio_item.portfolio_id = @to_portfolio.id
```

Comment:
Either line 27 or 45 is enough. No need both. Since we are doing `new_portfolio_item.save` next, removing line 27 is a better choice.